bionic/cache.py

```diff
@@ -888,7 +888,7 @@ def valid_filename_from_query(query):
     return query.dnode.to_descriptor().replace(" ", "-")
 
 
-CACHE_SCHEMA_VERSION = 6
+CACHE_SCHEMA_VERSION = 7
```

Comment:
Now that we have release notes, we can include this compatibility change in them.